### DISCUSSION TITLE
Open directories with a third party file manager

### DISCUSSION THREAD

**Discussion by sim186** [Flow-Launcher/Flow.Launcher#729] [Ideas]
Hi,

I just started to used flow launcher. Is that possibile to open a directory after the search with a different file manager?

**Comment by onesounds** [comment]
insert file manager(e.g total commander) path and filename into plugin>program>customzied explorer.
(like 'C:\Program Files\GPSoftware\Directory Opus\dopus.exe')
if you use everything, plugin>everything>customized explorer too.
It still doesn't work windows index search (aka explorer plugin).

**Comment by jjw24** [comment]
Yep we can add that in for Explorer plugin like how Everything is done. Issue created here #730. @sim186 in the mean time like @onesounds suggested you can use the Everything plugin if you want, via `pm install everything`

**Comment by onesounds** [comment]
https://github.com/Flow-Launcher/Flow.Launcher/pull/782 done.
